ergo-chain-generation/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "ergo-chain-generation"
+version = "0.14.0"
+license = "CC0-1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+base16 = "0.2.1"
+blake2 = "0.9"
+byteorder = "1"
+ergotree-ir = { version = "^0.14.0", path = "../ergotree-ir", features = ["arbitrary"] }
+ergotree-interpreter = { version = "^0.14.0", path = "../ergotree-interpreter", features = ["arbitrary"] }
+ergo-lib = {version = "^0.14.0", path = "../ergo-lib"}
```

Comment:
`ergo-lib` re-exports `ergotree-ir` and `ergotree-interpreter` in https://github.com/ergoplatform/sigma-rust/blob/92fb1e3a3b7e440734bb08b7b1f1502c8fb646b3/ergo-lib/src/lib.rs#L34-L35
The reason is to avoid dependency on versions different than `ergo-lib` is using and to reduce `Cargo.toml` edits on version updates. Think of `ergo-lib` as a kind of "umbrella" library for almost all our crates.
Could you please use re-exported types instead?

Reply:
Sorry about that, I think I've made this mistake before.

Reply:
Oh, no worries. It's no problem at all.